README.md

```diff
@@ -1,9 +1,59 @@
-# Testing
+# STAND-CONTROL-BOT
 
-### LDAP/AD server
+Simple and opinionated web server to facilitate the reuse of development servers and infrastructure. 
 
-It is preferable to use real AD server to run auth tests, since auth is designed to
+# Deployment
+
+[Releases](https://github.com/udv-group/stand-control-bot/releases) page has two prebuild binaries for Linux: `stand-control-bot` which includes telegram integration and `sever` which just runs a web server.
+
+App is expected to be deployed in the private network. It is recommended to use a reverse proxy (like Nginx) to enable TLS and Docker to run a web server. For integration with telegram also network access
+to telegram servers is required.
+
+Web server is using LDAP to manage auth and is designed to be used with AD servers. 
+
+## Configuration
+
+### Env
+
+`APP_ENVIRONMENT` - what environment app is running in. Possible values are `local` (default) and `production`. Affects configuration lookup.
+
+`CONFIG_DIR` - path to directory with configuration files, default is `/etc/stand-control-bot` for `production` environment.
+
+`INCLUDE_SPAN_EVENTS` - flag to include span events in log output, making it much more verbose. Possible values are `true` and `false` (default).
+
+`TELOXIDE_TOKEN` - telegram bot token to enable telegram notifications. Can be obtained from [@Botfather](https://t.me/botfather) bot on telegram.
+
+### Configuration files
+
+Consult [base.toml](configuration/base.toml) for a list of all possible configuration options.
+
+Both `base.toml` and `production.toml` (or `local.toml` for `APP_ENVIRONMENT=local`) need to be present in `CONFIG_DIR`.
+
+Configuration presidence is as follows: `env` > `production.toml`/`local.toml` > `base.toml`. 
+
+
+# Development
+
+Prerequisites:
+- Up to date Rust compiler ([rustup](https://www.rust-lang.org/tools/install) is strongly recommended)
+- Postgres 
```

Comment:
Version of the postgres must be 13 or higher 
https://github.com/udv-group/stand-control-bot/blob/main/migrations/20240401005014_init.sql#L6 `gen_random_uuid()` procedure 

https://www.postgresql.org/docs/13/functions-uuid.html

```suggestion
- Postgres (>=13)
```